test/acceptance/PushPullNotificationsSpec.scala

```diff
@@ -16,8 +16,6 @@
 
```

Comment:
Function below look wrong - `makeAPICall` will also return a Future - so you will get a Future wraped inside a Future. If you wait on the outer Future, the inner one will be independent. I am not sure if this is what you want.
```
  private def sendNotificationToCustomsNotificationFor(client: Client): Future[ExpectedCall] = {
    Future.successful {
      val expectedCall = createARandomExpectedCallFor(client)
      makeAPICall(expectedCall)
      expectedCall
    }
  }
```

Reply:
makeAPICall does not return a Future

Reply:
it is  def makeAPICall: (ExpectedCall => Unit)

Reply:
OK